src/networking/protocol.cpp

```diff
@@ -298,16 +298,22 @@ void PacketManager::ExecParseMessage(Packet *pkt, ResponseBuffer &responses) {
     responses.push_back(std::move(response));
     return;
   }
+
+  // For empty queries, just send parse complete response
+  if (query_string == "") {
+    skipped_stmt_ = true;
+    std::unique_ptr<Packet> response(new Packet());
+    response->msg_type = '1';
```

Comment:
We should also make the message types be constants in types.h instead of just putting '1'